packages/devextreme/js/__internal/ui/m_validation_engine.ts

```diff
@@ -778,22 +779,21 @@ const ValidationEngine = {
   },
 
   registerValidatorInGroup(group, validator) {
-    const groupConfig = ValidationEngine.addGroup(group);
+    const groupConfig = ValidationEngine.addGroup(group, true);
```

Comment:
[minor]
Let's instead of this add a [default parameter value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) for `isRemovable` parameter in `addGroup` function
```suggestion
    const groupConfig = ValidationEngine.addGroup(group);
```

Reply:
[913f413](https://github.com/DevExpress/DevExtreme/pull/27435/commits/913f41339e3295625f6fa15bc9fb8a0f1abcdb35)

Reply:
I don't know if I'm missing something or it really causes an error
![image](https://github.com/DevExpress/DevExtreme/assets/133762589/75bf3077-5309-4c06-b532-edb269570eea)
![image](https://github.com/DevExpress/DevExtreme/assets/133762589/4a3c4084-09fd-412b-bf15-449628c76496)

Reply:
That's because of this: https://github.com/DevExpress/DevExtreme/pull/27435#discussion_r1646386039.

Reply:
[a3d3b51](https://github.com/DevExpress/DevExtreme/pull/27435/commits/a3d3b51763ff995849707aac971fb31bb597c4d3)